### PR TITLE
pgs 19-20/297

### PR DESCRIPTION
**Pull request by steffantucker**

### CODE REVIEW

**Comment by gnomon-** [comment]
Thank you for all the effort you put into perfect character alignments.  Damn fine precision work.

File: furbysource.asm [hunk 1]
```diff
@@ -384,3 +384,125 @@ Pos_sen            EQU   02H  ;C1 - moto______ical sensor (intt C1)
...
A-16
+
A-17
+
A-18
+Stacktop     EQU   FFH   ;Stack Top
+
+
+;**********************************************************************
+****;**********************************************************************
```
Comment (gnomon-) [inline furbysource.asm]: I think this line might need a linebreak right before the `;` character

**Comment by steffantucker** [comment]
Sorry for the silence, been out of town. I've made the change to line 403, should be good to merge now. 
Thank you for noticing the precision, I think I spent more time making sure everything lined up the same as in the document than I did actually copying text.

**Comment by gnomon-** [comment]
That looks perfect, thank you so much!

**Pull request merged**